src/main/java/joules/Joules.java

```diff
@@ -0,0 +1,98 @@
+package joules;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+import joules.task.Deadline;
+import joules.task.Event;
+import joules.task.Task;
+import joules.task.TaskList;
+import joules.task.Todo;
+
+/**
+ * Entry point for the Joules chatbot application.
+ * <p>
+ * This class contains the {@code main} method that starts the chatbot,
+ * initializes stored tasks, and continuously processes user commands
+ * until the user exits with the {@code bye} command.
+ * </p>
+ */
+public class Joules {
+    /** Initial size of task list */
+    private static final int DEFAULT_CAPACITY = 100;
+
+    /** User interface for interacting with the chatbot */
+    private static final Ui UI = new Ui();
```

Comment:
This kind of relates to coding standard too, but I believe the variable UI should be in lowercase.
```suggestion
    private static final Ui ui = new Ui();
```

Reply:
Since I declared it static final (a constant), I used uppercase for it